packages/node-core/src/db/db.module.ts

```diff
@@ -66,6 +66,7 @@ const buildSequelizeOptions = (nodeConfig: NodeConfig, option: DbOption): Sequel
     dialect: 'postgres',
     ssl: nodeConfig.isPostgresSecureConnection,
     dialectOptions: {
+      options: {requestTimeout: nodeConfig.timeout * 1000}, // configure timeout is in seconds, requestTimeout is in milliseconds
```

Comment:
I don't know if its a good idea to apply a broad timeout for all queries. This could break certain operations such as rewind.

It also has other side effects as the timeout config is applied to a lot of other things. Increasing the timeout for a query might cause other things to timeout that shouldn't timeout and vice versa